set.mm

```diff
@@ -645322,6 +645405,65 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
     $( $j usage 'nnmulcom' avoids 'ax-mulcom'; $)
   $}
 
+  ${
+    mvrlsubd.a $e |- ( ph -> B e. CC ) $.
+    mvrlsubd.b $e |- ( ph -> C e. CC ) $.
+    mvrlsubd.1 $e |- ( ph -> A = ( B - C ) ) $.
+    $( Move RHS left subtraction to LHS.  Converse of ~ mvlraddd .
+       (Contributed by SN, 21-Aug-2024.) $)
+    mvrlsubd $p |- ( ph -> ( A + C ) = B ) $=
+      ( caddc co cmin cc subcld eqeltrd addcld pncand eqtrd subcan2d ) ABDHIZCD
+      ABDABCDJIZKGACDEFLMZFNEFARDJIBSABDTFOGPQ $.
+  $}
+
+  ${
+    laddrotrd.a $e |- ( ph -> A e. CC ) $.
+    laddrotrd.b $e |- ( ph -> B e. CC ) $.
+    laddrotrd.1 $e |- ( ph -> ( A + B ) = C ) $.
+    $( Rotate the variables right in an equation with addition on the left,
```

Comment:
I find "rotate" a bit strange. Why not:
> Move a left-addition in the LHS to a subtraction in the RHS, and then use symmetry of equality.

Reply:
It's easier for my brain to process it as a "rotation" rather than some two step combination (the comment notes two such combinations)

The purpose of consistency is easier understanding and use, so I realize it's a slippery trade off

I could add aliases `mvlladdcomd` etc. as an experiment? (No "usage is discouraged" since that would obviate the experiment, though)

Reply:
Mathbox, so I won't insist, though I find it very strange to "rotate" across an equality sign (at least in the present case).